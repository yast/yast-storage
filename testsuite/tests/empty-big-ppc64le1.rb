# encoding: utf-8

# testedfiles: helper1b.yh
module Yast
  class Empty1Client < Client

    def main
      Yast.include self, "setup-system.rb"

      setup_system("empty-big-ppc64le")

      Yast.include self, "helper1a.rb"

      Yast.import "ProductFeatures"

      ProductFeatures.SetBooleanFeature("partitioning", "try_separate_home", false)
      ProductFeatures.SetBooleanFeature("partitioning", "proposal_lvm", false)
      ProductFeatures.SetBooleanFeature("partitioning", "proposal_snapshots", false)
      ProductFeatures.SetStringFeature("partitioning", "vm_desired_size", "30 GB")
      ProductFeatures.SetStringFeature("partitioning", "root_base_size", "20 GB")

      Yast.include self, "helper1b.rb"

      nil
    end

  end
end

Yast::Empty1Client.new.main
