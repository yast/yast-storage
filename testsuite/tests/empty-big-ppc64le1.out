Dump	Proposal:
Dump	Create PPC PReP volume /dev/sda1 (7.84 MiB)
Dump	Create swap volume /dev/sda2 (517.72 MiB)
Dump	Create root volume /dev/sda3 (2.50 TiB) with btrfs
Dump	Create subvolume boot/grub2/power-ieee1275 on device /dev/sda3
Dump	Create subvolume home on device /dev/sda3
Dump	Create subvolume opt on device /dev/sda3
Dump	Create subvolume srv on device /dev/sda3
Dump	Create subvolume tmp on device /dev/sda3
Dump	Create subvolume usr/local on device /dev/sda3
Dump	Create subvolume var/crash on device /dev/sda3
Dump	Create subvolume var/lib/mailman on device /dev/sda3
Dump	Create subvolume var/lib/named on device /dev/sda3
Dump	Create subvolume var/lib/pgqsl on device /dev/sda3
Dump	Create subvolume var/log on device /dev/sda3
Dump	Create subvolume var/opt on device /dev/sda3
Dump	Create subvolume var/spool on device /dev/sda3
Dump	Create subvolume var/tmp on device /dev/sda3
Dump	
Dump	Target Map Excerpt:
Dump	device:/dev/sda label:gpt
Dump	device:/dev/sda1 fsid:0x108
Dump	device:/dev/sda2 fsid:0x82
Dump	device:/dev/sda3 fsid:0x83
